extensions/amp-story-dev-tools/0.1/amp-story-dev-tools.css

```diff
@@ -18,10 +18,87 @@ amp-story-dev-tools {
   width: 100% !important;
   height: 100% !important;
   box-sizing: border-box !important;
-  background-color: #121212 !important;
+  background-color: black !important;
   font-size: 16px !important;
   font-family: 'Poppins', sans-serif !important;
   color: white !important;
   padding: 30px 10% !important;
   text-align: center !important;
 }
+
+.i-amphtml-story-dev-tools-container {
+  width: 100% !important;
+  height: 100% !important;
+}
+
+.i-amphtml-story-dev-tools-header {
+  width: 100% !important;
```

Comment:
Might not need this since it's a block level element.

Reply:
True, same thing with the container above